lib/tapioca/generator.rb

```diff
@@ -415,27 +415,42 @@ def rbi_header(command, reason: nil, strictness: nil)
       [statement, sigil].compact.join("\n").strip.concat("\n\n")
     end
 
+    sig { returns(String) }
+    def empty_rbi_comment
+      statement = <<~CONTENT
+        # THIS IS AN EMPTY RBI FILE.
+        # see https://github.com/Shopify/tapioca/blob/master/README.md
+      CONTENT
+
+      statement
+    end
+
     sig { params(gem: Gemfile::Gem).void }
     def compile_gem_rbi(gem)
       compiler = Compilers::SymbolTableCompiler.new
       gem_name = set_color(gem.name, :yellow, :bold)
       say("Compiling #{gem_name}, this may take a few seconds... ")
 
       strictness = config.typed_overrides[gem.name] || "true"
-
+      rbi_body_content = compiler.compile(gem)
       content = String.new
       content << rbi_header(
         config.generate_command,
         reason: "types exported from the `#{gem.name}` gem",
         strictness: strictness
       )
-      content << compiler.compile(gem)
-
+      content << rbi_body_content
```

Comment:
This will again put multiple blank lines inside the file, which is what we are trying to avoid in the first place. Let's move this to the `else` block below.